website/content/docs/commands/var/purge.mdx

```diff
@@ -0,0 +1,63 @@
+---
+layout: docs
+page_title: "Command: var purge"
+description: |-
+  The "var purge" command permanently removes the specified secure variable
+  from Nomad.
+---
+
+# Command: var purge
```

Comment:
My one bikeshed is that `delete` feels more natural here and matches other product lines. I get that it matches `job stop purge` but that is maybe something we'd want to alias in a V2 so that we can have more RESTy nomenclature.

Reply:
We anticipate that there will be versions and soft-delete coming in a future release, so I selected `purge` for the current hard-delete behavior.